homeassistant/components/asuswrt/sensor.py

```diff
@@ -46,17 +50,44 @@ class AsusWrtSensorEntityDescription(SensorEntityDescription):
 
 UNIT_DEVICES = "Devices"
 
-CONNECTION_SENSORS: tuple[AsusWrtSensorEntityDescription, ...] = (
+ASUS_CPU_SENSORS: tuple[AsusWrtSensorEntityDescription, ...] = tuple(
+    AsusWrtSensorEntityDescription(
+        key=sens_key,
+        translation_key=sens_key,
+        icon="mdi:cpu-32-bit",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        suggested_display_precision=0,
+    )
+    for sens_key in SENSORS_CPU
+)
+ASUS_TEMP_SENSORS: tuple[AsusWrtSensorEntityDescription, ...] = tuple(
+    AsusWrtSensorEntityDescription(
+        key=sens_key,
+        translation_key=f"temp_{slugify(sens_key)}",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        suggested_display_precision=1,
+    )
+    for sens_key in SENSORS_TEMPERATURES
+)
+ASUS_SENSORS: tuple[AsusWrtSensorEntityDescription, ...] = (
     AsusWrtSensorEntityDescription(
         key=SENSORS_CONNECTED_DEVICE[0],
+        translation_key=SENSORS_CONNECTED_DEVICE[0],
         name="Devices Connected",
         icon="mdi:router-network",
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UNIT_DEVICES,
     ),
     AsusWrtSensorEntityDescription(
         key=SENSORS_RATES[0],
-        name="Download Speed",
+        translation_key=SENSORS_RATES[0],
```

Comment:
Translations are not related to protocol changes as this PR is trying to do. Please remove it from the PR.

Reply:
Moved to separate PR